### PR TITLE
Style fixes

### PR DESCRIPTION
**Pull request by jancborchardt**
Some small style fixes on top of the UI pull request. Please review @nickvergessen @owncloud/designers

### CODE REVIEW

**Comment by nickvergessen** [comment]
Do you really think the small red dot is enough?

File: css/styles.css [hunk 1]
```diff
@@ -80,7 +73,7 @@
 
 .notification .button {
     border: 1px solid rgba(240,240,240,.9) !important;
-    margin-left: 5px;
+    margin-left: 5px !important;
```
Comment (nickvergessen) [inline css/styles.css]: I hate important, why do we need it?

Comment (jancborchardt) [inline css/styles.css]: Because it’s otherwise overridden by the files app itself.

**Comment by jancborchardt** [comment]
> Do you really think the small red dot is enough?

Yeah, I also thought that. Increased the brightness of the red, and now the icon is displayed fully opaque (white) when there are new notifications.

**Comment by jancborchardt** [comment]
Folks, review please :) @owncloud/designers @stefan-niedermann @MorrisJobke @oparoz @ChristophWurst @Rekhyt

**Comment by PVince81** [comment]
Looks good so far.

Can you add explicit ".actions" styles for when not on the files list ?
See https://github.com/owncloud/notifications/issues/16

Currently it "inherits" the .actions styles from files.css.

**Comment by PVince81** [comment]
Note: the "x" button position (overlap) is fixed in this other PR: https://github.com/owncloud/notifications/pull/17

**Comment by jancborchardt** [comment]
Let's do that later, and merge now. Ok?

**Comment by PVince81** [comment]
Needs rebase

**Comment by nickvergessen** [comment]
@jancborchardt please dont merge stuff when tests did not pass, thanks

**Comment by jancborchardt** [comment]
While that is generally correct, CSS and images are not covered by tests, so they are completely irrelevant.

**Pull request merged**